dataland-frontend/package.json

```diff
@@ -35,27 +35,27 @@
   },
   "devDependencies": {
     "@cypress/code-coverage": "^3.10.0",
-    "@faker-js/faker": "^7.5.0",
+    "@faker-js/faker": "^7.6.0",
     "@types/big.js": "^6.1.6",
-    "@types/node": "^18.7.18",
-    "@vitejs/plugin-vue": "^3.1.0",
-    "@vue/eslint-config-typescript": "^11.0.1",
-    "@vue/test-utils": "^2.0.2",
+    "@types/node": "^18.11.0",
```

Comment:
In anderen Änderung switchst du das auf 18.10.0. Wieso hier auf 18.11.0?

Reply:
Das wurde automatisch generiert. 18.10.0 ist soweit ich gesehen habe die stable version. Würde dann hier entsprechend auch 18.10.0 einfügen.

Reply:
Geändert auf 18.11.0

Reply:
Also belässt du es auf 18.11.0 und machst überall anders auch 18.11.0?

Ist 18.11.0 nicht als stable markiert?

Reply:
Doch ist inzwischen als stabil markiert. Deshalb dann auch 18.11.0 überall